src/hooks/private/useFloatingTransition/useFloatingTransition.ts

```diff
@@ -55,6 +56,12 @@ export function useFloatingTransition({
     React.useEffect(() => {
         if (status === 'open' && previousStatus === 'initial') {
             onTransitionIn?.();
+
+            if (!enabled) {
+                requestAnimationFrame(() => {
```

Comment:
You must clear this effect

Reply:
What do you mean by clearing the effect, cancelling the frame?

Reply:
Yes

Reply:
To clear rAF on component unmount

Reply:
You suggest not to notify the user about transition completion if component was unmounted. I have the opposite opinion, if `onTransitionIn` called, then `onTransitionInComplete` must be called as well no matter what

Reply:
I just follow react docs guidance here

In strict mode this effect would be called twice, and i think this rAF would be called twice as result